specification/monitor/Azure.Monitor.Query.Logs/tspconfig.yaml

```
@@ -0,0 +1,50 @@
+parameters:
+  "service-dir":
+    default: "sdk/monitor"
+  "dependencies":
+    default: ""
+
+emit:
+  - "@azure-tools/typespec-autorest"
+
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    output-file: "../operationalinsights/{azure-resource-provider-folder}/Microsoft.OperationalInsights/{version-status}/{version}/OperationalInsights.json"
```

Do we need this `operationinsights` directory?
```suggestion
    output-file: "../{azure-resource-provider-folder}/Microsoft.OperationalInsights/{version-status}/{version}/OperationalInsights.json"
```